modules/graceful-restart.adoc

```diff
@@ -15,6 +15,29 @@ You can restart your cluster after it has been shut down gracefully.
 
 .Procedure
 
+. Turn on the control plane nodes.
++
+** If you are using the `admin.kubeconfig` from the cluster installation and the API VIP is up, complete the following steps:
```

Comment:
Hi Laura, Is the acronym VIP well known? I don't know what it is, but that's just me. :)

Reply:
@aspauldi Good question! It means virtual IP address. I will spell it out.